docs/tips/nix.md

```diff
@@ -0,0 +1,62 @@
+# Using on Nix/NixOS
+
+Emanote can be easily built using the Nix expressions provided in its source repo.
+
+You will need [Nix](https://nixos.org/download.html) version 2.4 or greater.
+
+## Installing with Nix Flakes
+
+This will provide the `emanote` command in your environment.
+
+```sh-session
+$ nix profile install github:srid/emanote
```

Comment:
TIL `nix profile install`.  Since Nix 2.4 got released I should probably use this for README.md too.